### ISSUE TITLE
Deployment idempotency

### ISSUE THREAD

**Issue by siwyd** [akamai/terraform-provider-akamai#226]
Deployment idempotency is missing since upgrading from pre 1.0 to 1.5:

```
  # akamai_property.dd_lfvp_stv will be updated in-place
  ~ resource "akamai_property" "dd_lfvp_stv" {
        id                 = "prp_593090"
      ~ latest_version     = 22 -> (known after apply)
        name               = "dd_lfvp_stv"
      ~ production_version = 22 -> (known after apply)
      ~ staging_version    = 21 -> (known after apply)
        # (8 unchanged attributes hidden)

        # (1 unchanged block hidden)
    }

  # akamai_property_activation.dd_lfvp_stv will be updated in-place
  ~ resource "akamai_property_activation" "dd_lfvp_stv" {
        id            = "prp_593090:PRODUCTION"
      ~ version       = 22 -> (known after apply)
        # (5 unchanged attributes hidden)
    }
```

When I do an apply, it 'changes' things, but nothing really happens. There is no activation or anything like that. How do I get idempotence again on my properties?

**Comment by umglurf** [comment]
We have the same issue, we have solved it with a workaround by setting in the akamai_property
```
lifecycle {
  ignore_changes = [latest_version, production_version, staging_version]
}
```
but this should not be needed.

**Comment by siwyd** [comment]
Would be nice to get an update on this. I'm not planning on upgrading anything to > 1.0 while this is still an issue so I can't help detect other issues while stuck on < 1.0.

**Comment by tiej-dr** [comment]
An upvote for this issue. The suggested lifecyle/ignore_changes workaround from @umglurf works fine, however it means that if there is a valid property change, the version diff is not detected by "akamai_property_activation" resource (setting version = latest_version). So does not work well if you want to activate every time property has been updated (in one single apply). Now our workaround is to run a manual plan/apply for the activation resource. Not sure if there is a better procedure for this, or we just need to wait for this issue to be fixed?

**Comment by siwyd** [comment]
It's a bit of a pity the provider is seeing little progress towards fixing bugs. To be honest, this project is not in an 1.0 state as far as I'm concerned. Things break very regularly, it's impossible to be a user of this provider without being willing to spend a lot of time debugging things.

**Comment by robertolopezlopez** [comment]
Hi @siwyd ,

This defect is fixed already in our internal repository, we will release the solution this month.

**Comment by lkowalsk-akamai-com** [comment]
Hi @siwyd 
This issue should be fixed now as the fix was released in v1.6.1

**Comment by siwyd** [comment]
@lkowalsk-akamai-com Indeed it is, cheers!
